### DISCUSSION TITLE
Live Listening Feature

### DISCUSSION THREAD

**Discussion by BenNeighbour** [matbroughty/timstwitterlisteningparty#19] [Ideas]
👋 Hi all, I think that having a feature where other listeners can listen together via the website itself could be very beneficial + you could run an AD at the bottom of that view or have ads in-between every few songs. So on the announcement tweets for any new events, you could have the link that the user can go, then that UI would have the picture of the album/name of the album/etc while playing the songs in order (just like in Spotify/Amazon Music). As well as this, on the same screen you could have chat/messaging functionality where people can chat directly on, rather than using Twitter. 

The implementation for something like wouldn't actually be too bad - you would just have to set up WebSockets and WebRTC going from some JS/TS client to either a Java/Kotlin Spring Boot Service that maintains the connection and allows the text/chat functionality too. If we do end up trying to implement this then I will do more research and provide more details as to how this could be done.

Would be cool to get anybody's ideas on this - maybe this would be a bit overkill and I do realize you are currently using Twitter as the main engine of this app which is maybe better. 🙂👍

**Comment by gingerbeardman** [comment]
Hi Ben, another Matt here. I take care of the shared calendar (iCloud webcal, but works with Google Calendar, etc) as well as the Twitter List and reservations system (TeamUp), manage the CSV, have some systems for checking whether the site and calendar are in sync, and I manually check album anniversary dates. My motivation when setting all of these up was to make good use of existing technology solutions rather than reinventing the wheel in any way.

Tim the founder is a big Twitter user and that's why he does this on there. Even so, there is some reluctance to try various other features offered by Twitter outside of the standard follow/timeline/retweet.

For example the Twitter List https://twitter.com/i/lists/1244975721304424448 can be viewed by anybody regardless of whether they are logged into twitter or not, through their web browser. This gives curated list of tweets relevant to each party. If you're logged into Twitter or the app, even better, as you can pin the List to your home page which gives an amazing user experience as you can swipe between the List and the hashtag and so on.

All good, but despite this exiting and improving the experience for those that know about it, there has been no desire to promote it from the management. I still update it almost every day, and it still 1.7K followers, but it sort of feels like a wasted effort in some ways.

Anyway, my point is the battle we face is communicating any idea higher up in order to get understanding and then build momentum behind it.
